web/source/kmwbase.ts

```diff
@@ -28,6 +28,20 @@
    Copyright 2017-2019 SIL International
 ***/
 namespace com.keyman {
+
+  export interface OptionType {
```

Comment:
😍

Reply:
One extra suggestion:

https://github.com/keymanapp/keyman/blob/1c1a9bcbb818e342f3c86940746124fad2f4d4c0/web/source/dom/domManager.ts#L1480

This is the function that consumes the new type.  May as well swap its type signature to use `OptionType`.